themes/ddbasic/template.node.php

```diff
@@ -194,10 +194,13 @@ function ddbasic_preprocess__node__ding_event(&$variables) {
 
       // Date.
       if (!empty($date)) {
-        $start_date = strtotime($date[0]['value']);
-        $end_date = strtotime($date[0]['value2']);
+        $date_start = new DateObject($date[0]['value'], new DateTimeZone($date[0]['timezone_db']));
+        $date_start->setTimezone(new DateTimeZone($date[0]['timezone']));
```

Comment:
Why is the timezone set two times?

Reply:
Why not do?: 
$date_start = new DateObject($date[0]['value'], new DateTimeZone($date[0]['timezone']));
Or?:
$date_start = new DateObject($date[0]['value']);
$date_start->setTimezone(new DateTimeZone($date[0]['timezone']));

Reply:
When the user saves the event time (e.g. danish time 2018-01-10 00:00), the value is saved in the database in UTC time (e.g. UTC time 2018-01-09 23:00). To print out the date/time properly we first need to create the dateObject with the UTC database time, and afterwards we can convert the dateObject db-time to localtime. Otherwise the DateObject dont know from what base time, to convert into local time.

Has been added as a comment into the PR.